BLMM/lib/fileio.py

```diff
@@ -517,3 +520,91 @@ def pracNumVoxelBlocks(inputs):
 
   # Return number of voxel blocks
   return(nvb)
+
+def cluster_detection(clustertype,maxmem):
+        # --------------------------------------------------------------------------------
+        # Set up cluster
+        # --------------------------------------------------------------------------------
+    with open(os.path.join(os.path.expanduser("~"), ".config/dask/jobqueue.yaml"), "r") as stream:
+        data = yaml.load(stream, Loader=yaml.FullLoader)
+    if maxmem < 21474836480:
+        maxmem = 21474836480
```

Comment:
We should also check if there is a memory limit in the users cluster configuration here. If the two conflict, we should probably give a warning and proceed with the user specified `MAXMEM`.